<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>559</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TEST LIB</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLCDNumber" name="BasculaLcd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>7</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
         <property name="value" stdset="0">
          <double>99999.979999999995925</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="BasculaEstable">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <disabledoff>:/devices/icons/system/devices/jockey-disabled.png</disabledoff>
           <disabledon>:/devices/icons/system/devices/jockey-enabled.png</disabledon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="semaforoRojo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/system.qrc">
            <normaloff>:/app/icons/app/alpr_off.png</normaloff>
            <normalon>:/app/icons/app/alpr_rojo.png</normalon>:/app/icons/app/alpr_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="semaforoAmarillo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/system.qrc">
            <normaloff>:/app/icons/app/alpr_off.png</normaloff>
            <normalon>:/app/icons/app/alpr_amarillo.png</normalon>:/app/icons/app/alpr_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="semaforoVerde">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources/system.qrc">
            <normaloff>:/app/icons/app/alpr_off.png</normaloff>
            <normalon>:/app/icons/app/alpr_verde.png</normalon>:/app/icons/app/alpr_off.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="camara1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>180</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>1280</width>
           <height>720</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="FotoMatriculaA">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>520</width>
             <height>110</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>520</width>
             <height>110</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="MatriculaA">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>MATRICULA A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="MatriculaPrecisionA">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>000%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="camara2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>180</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>1280</width>
           <height>720</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="FotoMatriculaB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>260</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>520</width>
             <height>110</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>520</width>
             <height>110</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="MatriculaB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>MATRICULA B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="MatriculaPrecisionB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>000%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/system.qrc"/>
 </resources>
 <connections/>
</ui>
